src/core/dagservice.ts

```diff
@@ -38,14 +38,15 @@ export interface ResolveResult {
   /**
    * remainderPath is the part of the path that wasn’t resolved yet.
    */
-  remainderPath?: string
+  remainderPath?: string;
```

Comment:
This is due to some incompatibility in some external deps... kinda a tricky one it seems.